.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,135 @@
+name: Benchmark
+
+on:
+  pull_request_review:
```

Comment:
Do we need a daily cron job for benchmark?

Reply:
Sure, it would be better to set up a daily benchmark.

While there are also other tasks that must be scheduled, e.g. sync-mainnet, integration-test. Therefore, I believe we should discuss these daily tasks together before writing them down. That's out of this PR.

Reply:
So, how to trigger `pull_request_review`? Could you add a brief doc here?

Reply:
Sure. I add one line comment in last commit 7d14ef2662877c11b20913012fa7d68e90e6bb6d: 

```
# Submit your review with a comment body containing "#benchmark"
```